cmd/virtual-workspaces/command/cmd.go

```diff
@@ -178,6 +179,13 @@ func Run(ctx context.Context, o *options.Options) error {
 		return err
 	}
 
+	admissionOptions := virtualoptions.NewAdmission()
```

Comment:
Should this already be a field on the Options? Initializing it here, empty, feels kind of wrong.

Reply:
We could, but there's nothing in the Admission struct at the moment (and it's probably going to stay that way), so I thought this is okay as well.